### PR TITLE
A bunch of static checker suggested improvements

### PR DESCRIPTION
**Pull request by M4rtinK**
As there are already some substantial changes in the Pykickstart 3 code base then it might make sense to also possibly also improve the code formatting when we are at it. :)

So I've run a battery of static checkers (Pylint, pyflakes, pep8, etc.) on the master branch and implemented the suggestions that looked sensible.

If some of the changes *don't* look sensible, let me know and I'll remove them from the PR. :)

### CODE REVIEW

File: pykickstart/commands/liveimg.py [hunk 1]
```diff
@@ -39,8 +39,8 @@ def __eq__(self, other):
             return False
 
         return self.url == other.url and self.proxy == other.proxy and \
-               self.noverifyssl == other.noverifyssl and \
-               self.checksum == other.checksum
+            self.noverifyssl == other.noverifyssl and \
+            self.checksum == other.checksum
```
Comment (clumens) [inline pykickstart/commands/liveimg.py]: I like all the conditions lined up to the right of the return here.  That occurs a handful of other places in this mega patch, too.  Aside from that everything looks great to me.

**Comment by jkonecny12** [comment]
Looks great to me too. Good job!

File: pykickstart/commands/liveimg.py (hunk 1 above)
Comment (M4rtinK) [inline pykickstart/commands/liveimg.py]: Should be changed now in all the places where it occurs. :)

**Comment by atodorov** [comment]
@M4rtinK - can you add your tools to `Makefile` and `.travis.yml` so they get run in CI as well ?

**Comment by M4rtinK** [comment]
@atodorov Well, its currently a bunch of vim plugins that that I use, so it is not really directly applicable. 

But I'm sure the plugins are just thin wrappers for the respective static checking utilities, which could then be called from the `Makefile` & CI. So I'll take a look what am I actually using. :)

**Pull request merged**